jobs/gtfs-schedule-validator/gtfs_schedule_validator.py

```diff
@@ -42,34 +43,25 @@
 #   similar to the extracts sharing some functionality
 class GTFSScheduleFeedValidation(PartitionedGCSArtifact):
     bucket: ClassVar[str] = SCHEDULE_VALIDATION_BUCKET
-    table: ClassVar[str] = "validation_reports"
-    extract: GTFSFeedExtractInfo = Field(..., exclude={"config"})
+    partition_names: ClassVar[List[str]] = GTFSScheduleFeedExtract.partition_names
+    table: ClassVar[str] = "validation_notices"
```

Comment:
ok, I think the actual validators call them notices, personally I think it could make sense to bite the bullet and switch to `validation_notices` now?

Reply:
yeah ok that's fine. and btw this is what sent me spiraling about the "backfill", because I was like oh is this going to cause problems in the bucket and then I realized that this table doesn't exist in the prod bucket at all  yet